### PR TITLE
Added Current News Section on Single News Page

### LINKED ISSUE #1555: [News] Add Related News Section on Single News page
**Current Behavior**
The single blog page has a Related Posts section at the bottom of the page. The single news page does not.

**Desired Behavior**
A Related New section should be presented the bottom of the single News page.

---
**Contributor Resources**
The `layer5` repo contains two websites. The current generation and the next-generation of the layer5.io site.

If the `layer5-ng` label is absent on this issue, then this issue pertains to the current generation of the layer5.io website, which uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [`layer5` contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

If the `layer5-ng` label is present on this issue, then this issue pertains to the next-generation of the layer5.io website, which uses Gatsby, Strapi, and GitHub Pages. Site content is found under the [`layer5-ng` branch](https://github.com/layer5io/layer5/tree/layer5-ng).
- See [`layer5-ng` contributing instructions](https://github.com/layer5io/layer5/blob/layer5-ng/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1555 

**Notes for Reviewers**
Screenshot_2021-04-06 Service Mesh Offers Promising Solution for Cloud Native Networking

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 5dd6a4678643902d5faaa168d37ebe0975f74efb at: https://606c7e19854a1f11039efc6c--layer5ng.netlify.app

File: src/sections/Blog/Related-NewsBlogs/relatedNewsBlogs.style.js [hunk 1]
```diff
@@ -0,0 +1,155 @@
+import styled from "styled-components";
```
Comment (leecalcote) [inline src/sections/Blog/Related-NewsBlogs/relatedNewsBlogs.style.js:1]: It'd be great to reuse as much of this as possible from the implementation in on the home page. Are these styles different?

Comment (Tanuj22) [reply src/sections/Blog/Related-NewsBlogs/relatedNewsBlogs.style.js:1]: Can you create this as a component in which only the required data would be passed as props? This component can the be reused in Related Blogs and News

Comment (chinmaym07) [reply src/sections/Blog/Related-NewsBlogs/relatedNewsBlogs.style.js:1]: Yes Correct .. I will do that .:+1:

**Comment by l5io** [comment]
🚀 Preview for commit 09c73a26463489db3c023c41023d7bfad50efa6b at: https://606dd035b976623438b5f9b6--layer5ng.netlify.app

File: src/sections/Blog/Related-Posts/index.js [hunk 2]
```diff
@@ -1,61 +1,21 @@
 import React from "react";
```
Comment (Tanuj22) [inline src/sections/Blog/Related-Posts/index.js:1]: Everthing looks good. Just move this to components and we are good to go

**Comment by l5io** [comment]
🚀 Preview for commit d98f2ddb42f377d8c7d8ea4716a80a756da914a9 at: https://606dec962b990400a432521c--layer5ng.netlify.app

**Pull request closed without merging**